apps/src/utils.js

```diff
@@ -406,7 +406,7 @@ export function showUnusedBlockQtip(targetElement) {
 /**
  * @param {string} key
  * @param {string} defaultValue
- * @return {string}
+ * @return {string} returns the value of the key in localStorage, null if not set or the defaultValue if there is an error
```

Comment:
Thanks for adding this documentation!

Reply:
I completely agree that this seems weird and if I were to use the local storage, I might make a new item that does use the default value if the item is not found. I added the extra documentation because it took me about an hour to find what was wrong because I made assumptions about the function before I finally went deeper into the MDN function.

I'm hoping that this was not an MDN function that changed after our implementation expected the old behavior, but since everything seems to be working correctly, it does not seem to be.